app/src/main/res/menu/toolbar_main_menu.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
```

Comment:
Hi  
Could you either fully implement or remove the search functionality?  
I do not like the idea of having unfinished features as the HEAD commit on master